conf/config.json

```diff
@@ -1,5 +1,16 @@
 {
-    "default_env": "us_sandbox",
+    "default_env": "localhost",
+    "localhost":{
+        "username" : "superadmin@zuora.com",
+        "password" : "123456",
+        "oauth_token" : "",
+        "rsa_signature": "/v1/rsa-signatures",
+        "zuora_base_url": "http://localhost:8080/apps",
+        "payment_page_url": "http://localhost:8080/apps/PublicHostedPageLite.do",
+        "pageId": "40282ccb86e8c2b00186e948b3cc0098",
+        "accountId":"4028ab2686e4eb5d0186e4f09a060238",
+        "publicKey": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkhYxg7O5D6JFAsDZMLMnawWiE4y2jrPKy91/x61JYGkCHLyvza273I+PFyT/y2Ya2nGei4jfSHLtWaOtuYUdBuqfW7k3YNobM0+pKan63Iciqw+UwDy+BeiuC6t4GbkQGMDXhU77DIOa/WnAl9hMtqlBIiP/ySZRiM+dp5Hhf8C9qOwHLwnhD/80ZgFjwstHbuQ1Yew4tHll9mhoRx86KOwKtni1fY6gjdeYDeoIWNR2djge6yHfLFAumTMNYw7yBDdwGAmYpoWBWuVcylAcWNFM/PRZcZ6ikcAxf7zTK6AZNneiBZk6i3f8OUcpQrvuQKbfdipdCc4g6fqEsbr21wIDAQAB"
+    },
```

Comment:
Let's remove this file's changes.

Reply:
Those are the changes made so that the local runs in the browser. If those are removed, the UI will not be opened

Reply:
@charvithaa we should not expose those confidenial information in the public repo of github. the user can follow README.md to update the config.json as you did.
Please remove those information.

Reply:
I have removed those, shall I close the ticket now